doc/data/messages/u/use-a-generator/details.rst

```diff
@@ -0,0 +1,3 @@
+By using a generator you can cut the execution tree and exit directly at the first element that is False or True instead of

```

Comment:
```suggestion
By using a generator you can cut the execution tree and exit directly at the first element that is ``False`` for ``all`` or ``True`` for ``any`` instead of

```